content/docs/user-guide/testing/getting-started/_index.md

```diff
@@ -0,0 +1,233 @@
+
+### Registering new C++ tests
+
+While reading the upcoming steps, refer to the following `CMakeLists.txt` example:
+
+```
+# Preexisting module registration from Step 0
+ly_add_target(
+    NAME MyModuleName
+    NAMESPACE Gem
+    FILES_CMAKE
+        mymodule_files.cmake
+    INCLUDE_DIRECTORIES
+        PRIVATE
+            Source
+        PUBLIC
+            Include
+    BUILD_DEPENDENCIES
+        PRIVATE
+            Legacy::CryCommon
+    COMPONENT
+        MyModuleComponent
+)
+
+# New test module registration from Step 1
+# this configures CMake to also build a library called MyModuleName.Tests.so (.dll on Windows)
+# the rest of CMake will refer to this freshly created module by its NAME ("MyModuleName.Tests")
+if(PAL_TRAIT_BUILD_SUPPORTS_TESTS)  # only create these test modules if the target platform supports testing
+
+    ly_add_target(
+        NAME MyModuleName.Tests SHARED
+        NAMESPACE Gem
+        FILES_CMAKE
+            mymodule_test_files.cmake
+        INCLUDE_DIRECTORIES
+            PRIVATE
+                Tests
+                Source
+        BUILD_DEPENDENCIES
+            PRIVATE
+                AZ::AzTest
+                Legacy::CryCommon
+                Gem::MyModuleName
+        COMPONENT
+            MyModuleComponent
+    )
+
+    # Register new test module from Step 2
+    ly_add_googletest(
+        NAME MyModuleName.Tests
+    )
+
+endif()
+```
+
+#### Step 0: Add a production build target
+
+Before configuring tests, you should first define the library which gets tested. This is defined in a `CMakeLists.txt`, which typically lives in the same directory as the code or in a parent directory. For more information on configuring `CMakeLists.txt`, please read more about adding library targets at [Getting Started with the CMake Build System](/docs/user-guide/build/). Note that you only need to define a library that will be built; you can start configuring and writing tests before you finish writing the production code.
+
+#### Step 1: Add a test build target
+
+Similar to the production build target, the test target defines a library in a `CMakeLists.txt` configuration file. Start by finding the `CMakeLists.txt` for your module, which should already be created from Step 0. It should exist at a path similar to `o3de/.../<MyModule>/CMakeLists.txt`.
+
+Modify the CMakeLists.txt file to define your new test module, using the helper function `ly_add_target()`. Similar to the production build target, it is easiest to create another file which lists the C++ files used to compile the test library. The example above uses `dev/.../<MyModule>/mymodule_test_files.cmake`, which would have content similar to:
+
+```
+set(FILES
+    test/MyModuleTestFile.cpp
+    test/MyModuleMathTests.cpp
+)
+```
+
+#### Step 2: Register the test module
+
+The final `CMakeLists.txt` step is to register the module with CTest. For this, use helper function `ly_add_googletest()`
```

Comment:
```suggestion
In `CMakeLists.txt`, register the module with CTest by using the helper function `ly_add_googletest()`.
```
You can change "final `CMakeLists.txt` step" to "In `CMakeLists.txt,` ..." which allows you to restructure the rest of the sentence so its more concise. Also, i think it's clearer to describe the step as taking place "In `CMakeLists.txt`".